client-templates/bitbucket-server/accept.json.sample

```diff
@@ -650,7 +659,18 @@
     {
       "//": "used to add Code Insights annotations to a report",
       "method": "POST",
-      "path": "/rest/insights/1.0/projects/:project/repos/:repo/commits/:sha/reports/:report/annotations",
+      "path": "/rest/insights/latest/projects/:project/repos/:repo/commits/:sha/reports/:report/annotations",
+      "origin": "https://${BITBUCKET}",
+      "auth": {
+        "scheme": "basic",
+        "username": "${BITBUCKET_USERNAME}",
+        "password": "${BITBUCKET_PASSWORD}"
+      }
+    },
+    {
+      "//": "get list of API capabilities for Code Insights",
+      "method": "GET",
+      "path": "/rest/insights/latest/capabilities",
```

Comment:
Added for
https://github.com/snyk/bitbucket-server-agent/blob/develop/lib/bitbucket-server/codeinsights.js#L109